modules/widgets/authors.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/*
+ * Widget to display blog authors with avatars and recent posts.
+ *
+ * Configurable parameters include:
+ * 1. Whether to display authors who haven't written any posts
+ * 2. The number of posts to be displayed per author (defaults to 0)
+ * 3. Avatar size
+ */
+class Widget_Authors extends WP_Widget {
```

Comment:
Should we prefix that class name to avoid conflicts with other plugins in the repository?